src/test/scala-2.12/org/apache/james/gatling/jmap/scenari/FeederJmapAllScenario.scala

```diff
@@ -6,16 +6,15 @@ import org.apache.james.gatling.jmap.scenari.common.Configuration._
 import org.apache.james.gatling.jmap.scenari.common.{CommonSteps, HttpSettings}
 import org.apache.james.gatling.jmap.{JmapMailboxes, JmapMessages}
 
-import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import scala.concurrent.duration._
 
 class FeederJmapAllScenario extends Simulation {
   private val loopVariableName = "any"
 
-  val users = csv("users.csv").circular.records
-    .map({ record => User(Username(record("username")), Password(record("password"))) })
-    .map(Future.apply(_))
+  val users = csv("users.csv").circular.readRecords
+    .map({ record =>    User(Username(record("username").toString), Password(record("password").toString)) })
```

Comment:
why these spaces ?
could we avoid calling toString on records?

Reply:
I tried something, please review

Reply:
a litte nitpicking but you could transform  the Username and Password class to value class in order to avoid boxing.
You just have to make them extend AnyVal.

Reply:
not really related to this PR, but done :)